src/main/scala/no/gdl/bookapi/model/domain/EditorsPick.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Part of GDL book_api.
+ * Copyright (C) 2017 Global Digital Library
+ *
+ * See LICENSE
+ */
+
+package no.gdl.bookapi.model.domain
+import no.gdl.bookapi.BookApiProperties
+import scalikejdbc._
+
+case class EditorsPick (id: Option[Long],
+                        revision: Option[Int],
+                        language: String,
+                        translationIds: Seq[Long])
```

Comment:
Just to be sure: Will this be an empty list if there are no values in the column? Does `BIGINT ARRAY` allow for `null`, or will it just be an empty list/array?

Reply:
Nice catch! in DB this will be `NULL`, and it explodes when trying to read a row without this set 💣 

Will change the DB-definition to not allow null-values, so at least it will be an empty list. (When creating a CRUD-frontend we will validate that you have to pick at least one book)